<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/163MusicAssistant.png</normaloff>:/image/163MusicAssistant.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelMusicName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ADMUI3Lg</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSingerName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ADMUI3Lg</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlbumName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ADMUI3Lg</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDownloadAddress">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ADMUI3Lg</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="buddy">
     <cstring>lineEditDownloadAddress</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDownloadAddress">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCopy">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMusic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSinger">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlbum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputSongID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ADMUI3Lg</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="buddy">
     <cstring>lineEditSongID</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSongID">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonParse">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDownload">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>70</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDownloadTo">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonParseAndDownload">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicture">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_option">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="action_about_software">
   <property name="text">
    <string>此软件...</string>
   </property>
  </action>
  <action name="action_about_me">
   <property name="text">
    <string>我</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditSongID</tabstop>
  <tabstop>pushButtonParse</tabstop>
  <tabstop>pushButtonCopy</tabstop>
  <tabstop>pushButtonDownload</tabstop>
  <tabstop>lineEditDownloadAddress</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
